etl/src/main/java/org/pih/hivmigration/etl/sql/TableStager.groovy

```diff
@@ -0,0 +1,60 @@
+package org.pih.hivmigration.etl.sql
+
+import org.apache.commons.dbutils.handlers.MapListHandler
+
+import java.util.stream.Collectors;
+
+class TableStager extends SqlMigrator {
+
+    String oracleTableName
+    String mysqlTableName
+
+    TableStager(String oracleTableName) {
+        this.oracleTableName = oracleTableName
+        this.mysqlTableName = "hivmigration_" + oracleTableName.replace("HIV_", "").toLowerCase()
+    }
+
+    @Override
+    void migrate() {
+        def datatypeMap = [VARCHAR2: "TEXT", NUMBER: "NUMERIC"]
```

Comment:
Why map varchar2 to text?  I would think we could construct an actual varchar column of the appropriate length with a bit more code.  If there is no performance or other negative impact to using text, I guess that's fine, but I would assume there is.

Reply:
The only negative impact of using TEXT is that you can't index on it. Is that okay?

Reply:
https://stackoverflow.com/a/25301046/1464495